src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -21,67 +21,67 @@ public static partial class SyntaxFactory
         /// <summary>
         /// A trivia with kind EndOfLineTrivia containing both the carriage return and line feed characters.
         /// </summary>
-        public static readonly SyntaxTrivia CarriageReturnLineFeed = Syntax.InternalSyntax.SyntaxFactory.CarriageReturnLineFeed;
+        public static SyntaxTrivia CarriageReturnLineFeed { get; } = Syntax.InternalSyntax.SyntaxFactory.CarriageReturnLineFeed;
```

Comment:
This could be a regular (non-autoprop) property since it's just referring to another (internal) field.
i.e.
```C#
public static SyntaxTrivia CarriageReturnLineFeed => Syntax.InternalSyntax.SyntaxFactory.CarriageReturnLineFeed;
```